### ISSUE TITLE
Unfortunately it doesn't work with two devices

### ISSUE THREAD

**Issue by Tomtom483** [shivan/homeassistant-liquidcheck#4]
I have an installation with two liquid checks. Unfortunately, the data is only retrieved from one device. Each of the two devices works on its own without any problems.

**Comment by shivan** [comment]
Was it added with two names? I did not try it yet with multiple devices.

**Comment by Tomtom483** [comment]
Yes, I tried different names. When setting up the second device, only the IP address is requested first.
![Screenshot 2023-12-15 060356](https://github.com/shivan/homeassistant-liquidcheck/assets/125215231/b74121af-0fda-40f3-bd79-3d924de96898)

**Comment by GUHU0433** [comment]
I'll have also two of them. It would be great, if this integration will support two devices.

**Comment by Phoenix-DH** [comment]
@shivan 
There is a download error: no manifest file found

**Comment by shivan** [comment]
How did you install it?

I go to HACS, add custom repository (since liquid-check is no more visible)

![Image](https://github.com/user-attachments/assets/ebf9e3c6-4850-41ab-b113-938397fb9b7a)

Then I can search for it in HACS and install 1.0.4.

![Image](https://github.com/user-attachments/assets/f538c253-81d9-4ef2-8964-5392547debd7)

**Comment by shivan** [comment]
It might be because I can see in your screenshot "liquid-check" but the folder is "liquid_check".
But I think there was something either - is not allowed or _ ...

**Comment by shivan** [comment]
renamed it and re-released 1.0.4

Next question - does it now work with two devices? Since I only had one. I tried some tricks to simulate a second device... would be nice to know, if it works with real two devices.

**Comment by Phoenix-DH** [comment]
Don‘t know. I haven‘t modified anything.

As mentioned, successfully updated.

I only have one.

**Comment by GUHU0433** [comment]
Thanks a lot for your work .. I use in the meantime plain rest-device in HA. Works also well.

My second liquid check is not working now ... I can check it later on in spring.

**Comment by shivan** [comment]
@GUHU0433 did you try 2.0.x already? Since multi device now should work, I'm going to close this issue.
